upload/upload.go

```diff
@@ -54,41 +53,63 @@ func loadConfigFile(path string) (*Config, error) {
 
 	cfg, err := ini.Load(path)
 	if err != nil {
-		return config, nil
+		return config, err
 	}
 
 	if err := cfg.Section("").MapTo(config); err != nil {
 		return nil, err
 	}
 
+	if config.AccessKey == "" || config.SecretKey == "" || config.AccessToken == "" {
```

Comment:
`SecretKey` is strictly not necessary since authentication is done with the token and `Accesskey` should be the username